### DISCUSSION TITLE
Uhrzeit / Zeitzone falsch

### DISCUSSION THREAD

**Discussion by MatzeF1909** [flobz/psa_car_controller#796] [Q&A]
Moin,

ich habe nun zum besseren späteren Update den PSA Car Controller über Portainer als Container installiert.
Davor hatte ich es als .venv auf meinen Raspberry Pi 5 laufen.
Ich habe heute festgestellt, dass die Zeiten im Log bzw. auf dem Webinterface um eine Stunde differieren.
Der letzte Akkustand vom Auto war um 8 Uhr, im PSA Car Controller wird mir aber 7 Uhr angezeigt.
Wie kann ich das lösen?  Muss ich Verzeichnisse vom Container und dem Host verbinden? Wenn ja, welche?
Ich bin dankbar für jede Hilfe.
Viele Grüße
Matze

**Comment by MatzeF1909** [comment]
Ich habe es mit den folgenden beiden Befehlen direkt auf der Container Console gelöst:
rm /etc/localtime
ln -s /usr/share/zoneinfo/Europe/Berlin /etc/localtime
Container neu gestartet und schon waren die Zeit im Log richtig
Ich fürchte nur, dass beim nächsten Update die Einstellung weg sind und ich die Befehle erneut ausführen muss.
Wenn also jemand eine Idee hat, wie ich dauerhaft die Zeitzone richtig einstelle, bin ich für jeden Hinweis dankbar.
Viele Grüße
Matze

**Reply by seb585** [reply]
Hinzufügen der Environment Variable TZ=Europe/Berlin scheint bei mir zu funktionieren
